doc/ubuntu_release_process.md

```diff
@@ -115,46 +117,50 @@ Last, we need to push our tag above.
 
 
 ### cherry-picked changes ###
-This is generally not the mechanism that is preferred for any release supported by trunk.  It may be used in order to get a upload in *really quickly* for a hot fix.
+This is generally not the preferred mechanism for any release supported by trunk.  It is used either to get a upload in *really quickly* for a hot fix or during release **Feature Freeze** when only bugfixes are FeatureFreezeExceptions (FFEs) are accepted.  The tool for doing this is in ``uss-tableflip/scripts/cherry-pick``.  It takes as import a commit-ish that it will create a patch to apply the commitish via debian/patches/cpick-\*.
 
 #### Cherry-pick Process
-The tool for doing this is in ``uss-tableflip/scripts/cherry-pick``.  It takes as import a commit-ish that it will create a cherry-pick from.
-
-    ## just tag current so you have a revert point.
-    $ git tag --delete xx-current >/dev/null 2>&1; git tag xx-current
-    $ git checkout ubuntu/xenial
-    $ cherry-pick dc2bd79949
-
-The tool will:
-
-  * add a new file to debian/patches/ named cpick-<commit>-topic that is dep-8 compatible
-  * add that patch to debian/patches/series
-  * verify that the patch applies (quilt push -a)
-  * give you a chance to edit formatted the debian/changelog entry
-  * prompt you to commit the change.
+```
+  $ git fetch upstream; git checkout upstream/ubuntu/xenial -b ubuntu/xenial
+  $ cherry-pick <new_cherry_pick_commitish>
+  $ cherry-pick <commit_hash_2>
+  # Put up two PRs for review ubuntu/$release and ubuntu/daily/$release
+  $ git push <your_remote> ubuntu/$release
+  $ git push <your_remote> ubuntu/daily/$release
+```
+
+
+The `cherry-pick` tool will:
+
+  * add a new file to debian/patches/ named cpick-<commit>-topic that is
+    dep-3 compatible
+  * append file to debian/patches/series
+  * calls quilt push -a
+  * prompts for commiting the debian/changelog
+  * checkout upstream/ubuntu/daily/$release branch
+  * merge ubuntu/$release into ubuntu/daily/$release
```

Comment:
Can we discuss why and what merging ubuntu/$release into ubuntu/daily/$release is doing?

As I understand the details, ubuntu/daily/$release would only need the cherry-pick commits from ubuntu/$release and then revert each one of those commits.

Why do we merge?  Can you walk me through an example/sequence where not merging ubuntu/$release into ubuntu/daily/$release has issues?

Reply:
This conversation is no longer applicable because we get reset --hard ubuntu/devel from ubuntu/daily/devel, so the cherry-pick tool doesn't need to merge. Only revert cpick files after the reset --hard occurs.